src/main/java/com/catapi/controller/TranslationController.java

```diff
@@ -38,4 +42,22 @@ public void updateTranslationManually(@PathVariable Long catFactTranslationId, @
     public String translateTest(@RequestParam String textToTranslate, @RequestParam Locale locale) {
         return translationService.translate(locale, textToTranslate);
     }
+
+    @PostMapping("/linguatools/translateAllBreeds")
+    public ResponseEntity<String> translateAllBreedsByLinguatools(@RequestParam Locale locale) {
+        try {
+            if (EnumSet.of(Locale.UK, Locale.RU).contains(Locale.valueOf(locale.toString()))) {
+                return ResponseEntity.ok(translationService.translateAllBreedsByLinguatools(locale));
+            } else {
+                throw new IllegalArgumentException(STR."Can not get translation with using parameter \{locale}");
+            }
+        } catch (IllegalArgumentException e) {
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(e.getMessage());
+        }
+    }
```

Comment:
You don't need to

1. return ResponseEntity<String>
2. check locale
3. put logic in controllers

Reply:
Fixed